dashboard/test/ui/features/contractEditorSections.feature

```diff
@@ -19,6 +19,7 @@ Scenario: Examples should be hidden when specified in contract editor
   And the "Definition" contract editor header is visible
 
   And I press "modalEditorClose"
+  And I press the last button with text "Ignore"
   And I configure the contract editor to disable examples
```

Comment:
1. Should we just configure the contract editor to disable examples _before_ we press "modalEditorClose"?
2. Does this modal animate on close, and/or should we use "And I close the dialog" for this step as well? (It should apply to anything with an x-close button)
3. Could these scenarios ever fail to show that pop-up, as they seemed to do on my local machine?  Should we fail in that case?